ansible/group_vars/vagrant.yml

```diff
@@ -10,3 +10,8 @@ private_interface: ansible_eth1
 # local_interface: ansible_lo
 
 kv_store: etcd
+
+## Port details
+etcd_client_port: 2379
+docker_port: 2375
+swarm_port: 3376
```

Comment:
Use `swarm_port` as `3375`. 3376 can be used as swarm TLS port.

3375 (swarm) - 2375 (docker)
3376(swarm-TLS) - 2376 (docker-TLS)